selfdrive/car/honda/values.py

```diff
@@ -37,6 +39,11 @@ def __init__(self, CP):
     self.STEER_LOOKUP_V = [v * -1 for v in CP.lateralParams.torqueV][1:][::-1] + list(CP.lateralParams.torqueV)
 
 
+class Flag(IntFlag):
+  # Bosch models with alternate set of LKAS_HUD messages
+  BOSCH_EXT_HUD = 1
+
+
```

Comment:
suggestions:
  * use a name like `CarParamFlags`.  `Flag` is going to cause confusion in any module that happens to import `enum.Flag`.
  * derive from `Flag` rather than `IntFlag`.  This won't affect your existing code like `x |= BOSCH_EXT_HUD.value`.  `IntFlag` is intended more for use with legacy API's, letting you pass an `IntFlag` instance where an `int` is expected-- the downside is less type safety.

Reply:
After experimenting with this CP.flags approach, I found it limiting because Honda-specific details include non-flag values like STEER_THRESHOLD and SPEED_FACTOR.

Consider adopting #23053 instead, as it works generally across bool, number, string, list, etc.

Reply:
renamed to HondaFlags. I'll respond to your alternative approach on that PR